src/MediaWiki.ts

```diff
@@ -109,6 +109,30 @@ class MediaWiki {
     return `${this.apiUrl.href}action=query&list=categorymembers&cmtype=subcat&cmlimit=max&format=json&cmtitle=${encodeURIComponent(articleId)}&cmcontinue=${continueStr}`;
   }
 
+  public getVeApiArticleUrl(articleId: string): string {
+    return `${this.veApiUrl.href}${encodeURIComponent(articleId)}`;
+  }
+
+  public getDesktopRestApiArticleUrl(articleId: string): string {
+    return `${this.desktopRestApiUrl.href}${encodeURIComponent(articleId)}`;
+  }
+
+  public getMobileRestApiArticleUrl(articleId: string): string {
+    return `${this.mobileRestApiUrl.href}${encodeURIComponent(articleId)}`;
+  }
+
+  public getApiQueryUrl(query: string): string {
+    return `${this.apiUrl.href}${query}`
+  }
+
+  public getWebArticleUrlRaw(article: string): string {
+    return `${this.webUrl.href}${encodeURIComponent(article)}`
+  }
+
+  public getWebQueryUrl(article: string): string {
```

Comment:
Variable shoud be articleid, not article

Reply:
Code is wrong, ideally this method should be based on getwebqueryurl()

Reply:
I will remove this for now